docs/docs/user/references/editor-integration.md

```diff
@@ -0,0 +1,24 @@
+# Editor integration
+
+KPOps provides JSON schemas that enable autocompletion and validation for some
+of the files that the user must work with.
+
+## Supported files
+
+- [`pipeline.yaml`](/user/references/components)
+- [`config.yaml`](/user/references/config)
+
+## Usage
+
+1. Check that your editor implements the
+[yaml-language-server](https://github.com/redhat-developer/yaml-language-server#clients)
+2. Make sure to install the needed extension/plugin
+3. Add the text below to your editor's settings.
```

Comment:
imo the usage steps could be reduced and straight to the point. I am thinking something along the lines of:

1. Install the YAML language server in your editor of choice. (requires LSP support)
2. Configure the extension with the settings below

Reply:
true, fixed